services/director-v2/requirements/_base.txt

```diff
@@ -51,8 +51,10 @@ certifi==2021.5.30
     #   httpx
     #   requests
 chardet==4.0.0
+    # via aiohttp
```

Comment:
As we discussed many times, IMO i think is better to do upgrades in a separate PR to make sure any of these are adding some negative side effects. 
I leave the decision to you

Reply:
Thanks for pointing it out. You are my dependency 🦸‍♂️ . Think I managed to properly address it now